repository.datadog.yml

```diff
@@ -0,0 +1,7 @@
+---
+schema-version: v1
+kind: mergequeue
+team: ci-interfaces
+enable: true
+merge_method: merge
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we are enforcing 'squash' merges on the main branch:

```suggestion
merge_method: squash
```

Reply:
yeah, I removed the option already, the default for the repo will be used.